### ISSUE TITLE
bower Install Warnings and npm install errors

### ISSUE THREAD

**Issue by asifiqbal-ai** [seanjs-stack/seanjs#5]
[bower install msg.txt](https://github.com/seanjs-stack/seanjs/files/34830/bower.install.msg.txt)

I am attaching Bower Warning as above. 

and I am also attaching the log of NPM install. I tried with different version of npm and different version of node but errors are same. I also tried to compile mean. It has almost same dependencies but it is okay. i dont know what is the issue with sean. Please have a look.. thanks in advance for you time.

[npm-debug.txt](https://github.com/seanjs-stack/seanjs/files/34831/npm-debug.txt)

**Comment by Massad** [comment]
About the bower log, it will not cause a conflict.

Can you please tell me if you installed the seanjs from `yo seanjs` or by cloning the code itself and running the commands `npm install && bower install`

Also, can you verify if the application works using `node server.js` or not.

**Comment by asifiqbal-ai** [comment]
[npm-debug.log.txt](https://github.com/seanjs-stack/seanjs/files/34903/npm-debug.log.txt)

yo generator also have the same error .. on both windows and linux. Attached is the log from Ubuntu.

**Comment by asifiqbal-ai** [comment]
[npm-debug.log.txt](https://github.com/seanjs-stack/seanjs/files/34904/npm-debug.log.txt)

Attached is the file from windows.

**Comment by Massad** [comment]
As i can see from the logs, the modules are already installed.

What's the error on `node server.js`?

**Comment by asifiqbal-ai** [comment]
After installing using yo on ubuntu. I am seeing blank screen as attached.

![screenshot from 2015-11-15 16 31 30](https://cloud.githubusercontent.com/assets/9269567/11172259/3f61b7d0-8bb9-11e5-9acc-c1e666e747dc.png)

After installing using git clone and manually run the commands. Node server.js finally showed me the screen but i have no authorization to PUT and POST. stylesheet is also not okay. here are the snapshots

![i can logon to system](https://cloud.githubusercontent.com/assets/9269567/11172271/860a7672-8bb9-11e5-891d-4cfeba74c189.png)

![no uathoriztion to access](https://cloud.githubusercontent.com/assets/9269567/11172278/a5e7530c-8bb9-11e5-8563-b227d687e73a.png)

On windows i have exactly same output.

**Comment by Massad** [comment]
For the first screenshot, you are missing the `bower install` or just send me the console error in the browser blank page. (It's a client-side error)

For the permissions, on which page you are trying to access?

**Comment by asifiqbal-ai** [comment]
Mostly with PUT and POST i have issue. I can create accounts but i cannot do much with those accounts.

**Comment by Massad** [comment]
This is strange... because you are the admin as I can see from the menu.

Can you see the users list in the admin? Can you edit your profile? If cannot, can you please verify the `roles` in the `users` table for the current user?

**Comment by asifiqbal-ai** [comment]
No.. I checked in database. database is correctly populated. the blank screen problem is gone after doing bower install manually on yo generator. Yes it is very strange. I have tried almost all possible solution. I have downgraded node to check whether this issue comes with new versions or not. but all have same output. I tried node 0, 1, 4, 4.2.2 with npm older version to newer version. mean with same configuration is working very fine. I am very confuse. :(
